spec.html

```diff
@@ -38390,6 +38412,86 @@ <h1>Array.prototype.toLocaleString ( [ _reserved1_ [ , _reserved2_ ] ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.prototype.toreversed">
+        <h1>Array.prototype.toReversed ( )</h1>
+        <p>This method performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _O_ be ? ToObject(*this* value).
+          1. Let _len_ be ? LengthOfArrayLike(_O_).
+          1. Let _A_ be ? ArrayCreate(_len_).
+          1. Let _k_ be 0.
+          1. Repeat, while _k_ &lt; _len_,
+            1. Let _from_ be ! ToString(𝔽(_len_ - _k_ - 1)).
+            1. Let _Pk_ be ! ToString(𝔽(_k_)).
+            1. Let _fromValue_ be ? Get(_O_, _from_).
+            1. Perform ! CreateDataPropertyOrThrow(_A_, _Pk_, _fromValue_).
+            1. Set _k_ to _k_ + 1.
+          1. Return _A_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-array.prototype.tosorted">
+        <h1>Array.prototype.toSorted ( _comparefn_ )</h1>
+        <p>This method performs the following steps when called:</p>
+        <emu-alg>
+          1. If _comparefn_ is not *undefined* and IsCallable(_comparefn_) is *false*, throw a *TypeError* exception.
+          1. Let _O_ be ? ToObject(*this* value).
+          1. Let _len_ be ? LengthOfArrayLike(_O_).
+          1. Let _A_ be ? ArrayCreate(_len_).
+          1. Let _SortCompare_ be a new Abstract Closure with parameters (_x_, _y_) that captures _comparefn_ and performs the following steps when called:
+            1. Return ? CompareArrayElements(_x_, _y_, _comparefn_).
+          1. Let _sortedList_ be ? SortIndexedProperties(_O_, _len_, _SortCompare_, ~read-through-holes~).
+          1. Let _j_ be 0.
+          1. Repeat, while _j_ &lt; _len_,
+            1. Perform ! CreateDataPropertyOrThrow(_A_, ! ToString(𝔽(_j_)), _sortedList_[_j_]).
+            1. Set _j_ to _j_ + 1.
+          1. Return _A_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-array.prototype.tospliced">
+        <h1>Array.prototype.toSpliced ( _start_, _deleteCount_, ..._items_ )</h1>
```

Comment:
Nit, but the use of `delete` in these aliases implies mutation. Maybe it would be better to use `skip` instead.

Reply:
have updated to use 'skip'